src/lib/components/banner/Banner.component.js

```diff
@@ -0,0 +1,80 @@
+//@flow
+import React from "react";
+import styled from "styled-components";
+import * as defaultTheme from "../../style/theme";
+import { mergeTheme } from "../../utils";
+const STATUS_BANNER_WARNING = "STATUS_BANNER_WARNING";
+const STATUS_BANNER_ERROR = "STATUS_BANNER_ERROR";
+
+type Props = {
+  icon: Object,
+  title: string,
+  messages: Array<string>,
+  type: string
```

Comment:
you should use `Variant` instead to be consistent with others components.

`import type { Variant } from "../constants";`